ext/node/polyfills/_fs/_fs_open.ts

```diff
@@ -125,29 +126,29 @@ export function open(
       if (err) {
         (callback as (err: Error) => void)(err);
       } else {
-        callback(null, res!);
+        callback(null, new FileHandle(res!));
```

Comment:
I think the callback API may actually want the number here. The node.js docs suggest this is the case.

Reply:
Yes, this API should use a number